bundles/framework/layerlist/resources/scss/style.scss

```diff
@@ -1,3 +1,6 @@
+$editIcon: url("../images/glyphicons-151-edit.png");
+$addSubThemeIcon: url("../images/glyphicons-191-plus-sign.png");
```

Comment:
Could you use AntD icons instead?

Reply:
These are what admin-layerlist already used, so either leave these in place or we need to change the icons for admin-layerlist too

Reply:
I don't understand this. What is this admin-layerlist? And why would you need to change them there if you use AntD icons here?

Reply:
*admin-layereditor, aka the admin functionalities of layerlist. These icons were already in use in there.

Reply:
Can you link where? I can't find any reference to png-files in admin-layereditor.

Reply:
Actually it was in admin-hierarchical-layerlist, but admin-leyereditor apparently uses the same icons anyways: https://demo.oskari.org/Oskari/dist/1.6.0-dev/geoportal/icons.png

Reply:
Thats the generic sprite that is generated on build. I'm pretty sure it doesn't include pngs referenced by bundle scss-files but only the icons in oskari-frontend/resources/icons. But unless you can point to where they are used in the current admin (admin-layereditor) I'd like them replaced by an SVG icon from the AntD icons.

Reply:
https://github.com/oskariorg/oskari-frontend/blob/9aa98238fbd49d4da2854ef7b71891420c69bd96/resources/icons.css#L137

https://github.com/oskariorg/oskari-frontend/blob/9aa98238fbd49d4da2854ef7b71891420c69bd96/bundles/admin/admin-layereditor/instance.js#L129

Reply:
Apparently the 'editIcon' was different icon than I originally thought but hasn't worked, the plus sign has. The admin-layereditor icons for editing the layer and theme (group) is an icon from the generated sprite and this 'addSubThemeIcon' icon is also an icon that is present in the sprite , just not taken from there in this instance. So changing these would mean we need to change the icon for 'edit-layer' too to keep consistency (?)

https://github.com/oskariorg/oskari-frontend/blob/9aa98238fbd49d4da2854ef7b71891420c69bd96/resources/icons.css#L137-L143

https://github.com/oskariorg/oskari-frontend/blob/9aa98238fbd49d4da2854ef7b71891420c69bd96/bundles/admin/admin-layereditor/instance.js#L129

Reply:
Note that the binaries for icons are still here with the style.scss changes that shouldn't be required anymore with the SVG icons. Remove these.